lib/connections/limiter.go

```diff
@@ -134,37 +222,30 @@ func (lim *limiter) CommitConfiguration(from, to config.Configuration) bool {
 
 	if from.Options.MaxRecvKbps == to.Options.MaxRecvKbps &&
 		from.Options.MaxSendKbps == to.Options.MaxSendKbps &&
-		from.Options.LimitBandwidthInLan == to.Options.LimitBandwidthInLan {
+		from.Options.LimitBandwidthInLan == to.Options.LimitBandwidthInLan &&
+		lim.timer == nil && !to.Options.ScheduledRatesEnabled {
```

Comment:
This does not return early, if scheduled rate limits are enabled both in from and to and no limits changed, but it could, right? I'd have expected this:

```suggestion
		from.Options.ScheduledRatesEnabled == to.Options.ScheduledRatesEnabled {
```

Am I missing something?

Reply:
I didn't want to return early in any case that timer should exist, since in this case we have to check 2 next values for scheduled limits and 4 next values for time changing.
```suggestion
		from.Options.ScheduledRatesEnabled == to.Options.ScheduledRatesEnabled &&
		from.Options.ScheduledRatesFromHour == to.Options.ScheduledRatesFromHour &&
		from.Options.ScheduledRatesFromMinute == to.Options.ScheduledRatesFromMinute &&
		from.Options.ScheduledRatesToHour == to.Options.ScheduledRatesToHour && 
		from.Options.ScheduledRatesToMinute == to.Options.ScheduledRatesToMinute &&
		from.Options.MaxScheduledSendKbps == to.Options.MaxScheduledSendKbps &&
		from.Options.MaxScheduledRecvKbps == to.Options.MaxScheduledRecvKbps {
```

Reply:
Ah that's what I missed - thanks for reminding me :)